public/app/plugins/datasource/graphite/FunctionEditor.tsx

```diff
@@ -1,16 +1,32 @@
 import React, { useRef } from 'react';
-import { PopoverController, Popover, ClickOutsideWrapper, Icon, Tooltip, useTheme } from '@grafana/ui';
+import { PopoverController, Popover, ClickOutsideWrapper, Icon, Tooltip, useStyles2 } from '@grafana/ui';
 import { FunctionEditorControls, FunctionEditorControlsProps } from './FunctionEditorControls';
 import { FuncInstance } from './gfunc';
 import { css } from '@emotion/css';
+import { GrafanaTheme2 } from '@grafana/data';
 
 interface FunctionEditorProps extends FunctionEditorControlsProps {
   func: FuncInstance;
 }
 
+const getStyles = (theme: GrafanaTheme2) => {
+  return {
+    icon: css`
+      margin-right: ${theme.spacing(0.5)};
+    `,
+    label: css({
+      fontWeight: theme.typography.fontWeightMedium,
+      fontSize: theme.typography.size.sm, // to match .gf-form-label
```

Comment:
Isn't `theme.typography.size.X` deprecated? I think `theme.typography.bodySmall.fontSize` might be the equivalent.

Reply:
Thanks! Updated in f017c527a8a90413d16bf44692736b705026d4d0